api/provider/aws/test_aws/environment_create_test.go

```diff
@@ -251,6 +157,7 @@ func TestEnvironmentCreateDefaults(t *testing.T) {
 
 	req := models.CreateEnvironmentRequest{
 		EnvironmentName: "env_name",
+		Scale:           1,
```

Comment:
Why set the Scale when it isn't relevant to the test?

Reply:
✅